contracts/truefi/TrueLender.sol

```diff
@@ -58,11 +58,11 @@ contract TrueLender is ITrueLender, Ownable {
     uint256 public minApy;
     uint256 public maxApy;
 
-    // How many votes in prediction market
-    uint256 public participationFactor;
+    // How many votes are needed for a loan to be approved
+    uint256 public minVotes;
 
-    // How much worse is it to lose $1 TUSD than it is to gain $1 TUSD
-    uint256 public riskAversion;
+    // Minimum ratio of yes votes to note votes for a loan to be approved
+    uint256 public minYesToNoRatio;
```

Comment:
rename this variable to minRatio and add a comment:
// minimum ratio of yes to total votes required to pass a loan
// basis precision: 10000 = 100%

Reply:
Add a constant at the bottom of storage
uint256 constant BASIS_RATIO = 10000